README.md

```diff
@@ -60,3 +60,53 @@ guide](https://github.com/dowjones/tokendito/blob/main/docs/README.md#multi-tile
 
 ### Tips, tricks, troubleshooting, examples, and more docs are [here]()https://github.com/dowjones/tokendito/blob/main/docs/README.md!
 [Contributions are welcome](https://github.com/dowjones/tokendito/blob/main/docs/CONTRIBUTING.md)!
+
+## Docker
+
+Using Docker eliminates the need to install tokendito and its requirements.
+
+### Building the container image
+
+``` bash
+docker image build --pull --tag "tokendito:latest" .
+```
+
+### Running the container image
+
+Run tokendito with the `docker run` command
+
+``` bash
+docker run tokendito --version
+```
+
+You must map a volume in the Docker command to allow tokendito to write AWS credentials to your local system for use.  This is done with the `-v` flag.  See [Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#-mount-volume--v---read-only) for help setting the syntax.  The following directories are used by tokendito and should be considered when mapping volumes:
+
+- `/home/appuser/.aws/` (AWS credential storage)
+- `/home/appuser/.config/tokendito/` (tokendito profile storage)
+
+These can be covered by mapping a single volume to both the host and container users' home directories (`/home/appuser/` is the home directory in the container and must be explicitly defined).  You may also map multiple volumes if you have custom configuration locations and require granularity.
+
+Be sure to set the `-ti` flags to enable an interactive terminal session.
+
+``` pwsh
+docker run -ti -v ${home}:/home/appuser/ tokendito
+```
+
+Tokendito command line arguments are supported as well.
+
+``` pwsh
```

Comment:
please change pwsh to bash

Reply:
If you want to standardize on bash over powershell, should I also reformat the command?  I don't believe the line wrap character in powershell works for bash

Reply:
Also I think `${home}` might be problematic outside of powershell but I'm not sure

Reply:
oh I merely wanted to use something standard and not mix OS flavors in the doc. bash is not correct either, I put that out of a bad habit! Maybe put 'sh' or 'txt' instead.

Reply:
> Also I think `${home}` might be problematic outside of powershell but I'm not sure

yes, it would be ${HOME} in *nix, but it's clear you meant the home dir; you could rename it to ${your_home_directory}.